.travis.yml

```diff
@@ -5,7 +5,6 @@ perl:
 - 5.14
 - 5.12
 - 5.10
-- 5.8
```

Comment:
We no longer need this since the master is already working with perl 5.8 in traivs ci.